snapcraft/tests/test_common.py

```diff
@@ -23,6 +23,17 @@
 
 class CommonTestCase(tests.TestCase):
 
+    def setUp(self):
+        super().setUp()
+        self.addCleanup(self._reset_default_paths)
+
+    def _reset_default_paths(self):
```

Comment:
@elopio why not here? https://github.com/ubuntu-core/snapcraft/blob/master/snapcraft/tests/__init__.py#L50